python/ray/serve/_private/controller.py

```diff
@@ -212,6 +228,22 @@ async def __init__(
             description="The number of times that controller has started.",
         ).inc()
 
+    def reconfigure_system_logging_config(self, logging_config: LoggingConfig):
+        if self.logging_config.version == logging_config.version:
+            return
+        self.kv.put(LOGGING_CONFIG_CHECKPOINT_KEY, pickle.dumps(logging_config))
+        self.logging_config = logging_config
+
+        self.long_poll_host.notify_changed(
+            LongPollNamespace.SYSTEM_LOGGING_CONFIG,
+            logging_config,
+        )
+        configure_component_logger(
+            component_name="controller",
+            component_id=str(os.getpid()),
+            logging_config=logging_config,
+        )
```

Comment:
put this in a shared method that's also called in constructor (to avoid duplicating the PID and "controller" constant)

Reply:
put configure logger inside the same function.